src/content/en/tools/chrome-devtools/debug/remote-debugging/remote-debugging.markdown

```diff
@@ -1,168 +1,165 @@
 ---
 layout: shared/narrow
-title: "Remote Debugging Devices"
-description: "Web content behaves differently on mobile than on desktop. Debug live content on your device from your development machine using remote debugging."
+title: "Remote Debug Android Devices"
```

Comment:
"Remote Debugging" is our number one search term, so while I agree that "Remote debug" sounds better, I think we need to keep it in the title.